src/Microsoft.Diagnostics.Monitoring.WebApi/Exceptions/ExceptionsConfiguration.cs

```diff
@@ -0,0 +1,134 @@
+                {
+                    return true;
+                }
+            }
+
+            return false;
+        }
+
+        internal bool ShouldInclude(IExceptionInstance exception)
+        {
+            Func<ExceptionConfiguration, CallStackFrame, bool> evaluateFilterList = (configuration, topFrame) =>
+            {
+                bool include = true;
+                if (topFrame != null)
+                {
+                    CompareIncludeValues(configuration.MethodName, topFrame.MethodName, ref include);
+                    CompareIncludeValues(configuration.ModuleName, topFrame.ModuleName, ref include);
+                    CompareIncludeValues(configuration.ClassName, topFrame.ClassName, ref include);
+                }
+
+                CompareIncludeValues(configuration.ExceptionType, exception.TypeName, ref include);
+
+                return include;
+            };
+
+            return CheckConfiguration(exception, Include, evaluateFilterList);
+        }
+
+        internal bool ShouldExclude(IExceptionInstance exception)
+        {
+            Func<ExceptionConfiguration, CallStackFrame, bool> evaluateFilterList = (configuration, topFrame) =>
+            {
+                bool? exclude = null;
+                if (topFrame != null)
+                {
+                    CompareExcludeValues(configuration.MethodName, topFrame.MethodName, ref exclude);
+                    CompareExcludeValues(configuration.ModuleName, topFrame.ModuleName, ref exclude);
+                    CompareExcludeValues(configuration.ClassName, topFrame.ClassName, ref exclude);
+                    CompareExcludeValues(configuration.ExceptionType, exception.TypeName, ref exclude);
+                }
+                else
+                {
+                    CompareExcludeValues(configuration.ExceptionType, exception.TypeName, ref exclude);
+                }
+
+                return exclude ?? false;
+            };
+
+            return CheckConfiguration(exception, Exclude, evaluateFilterList);
+        }
+
+        private static void CompareIncludeValues(string configurationValue, string actualValue, ref bool include)
+        {
+            if (include && !string.IsNullOrEmpty(configurationValue))
+            {
+                include = CompareValues(configurationValue, actualValue);
+            }
+        }
+
+        private static void CompareExcludeValues(string configurationValue, string actualValue, ref bool? exclude)
+        {
+            if (exclude != false && !string.IsNullOrEmpty(configurationValue))
+            {
+                exclude = CompareValues(configurationValue, actualValue);
+            }
+        }
+
+        private static bool CompareValues(string configurationValue, string actualValue)
+        {
+            return actualValue.Contains(configurationValue, StringComparison.CurrentCultureIgnoreCase);
+        }
+    }
+
+    public class ExceptionConfiguration
+    {
```

Comment:
I believe there was discussion earlier about aligning this with `MethodDescription` IIRC - the way I have it here is designed to directly correlate to the `/exceptions` JSON payload for consistency and ease-of-use, which references methodName/className/moduleName. I could also see this growing in the future (for example, to include a list of parameter types), but I know we're starting simple for now

Reply:
I think standardization between the features would be good - what I was getting at is that, at the very least, I want `/exceptions` to be consistent with itself (hence why it's currently using module/class/method)